shell/app/locales/en.json

```diff
@@ -432,7 +431,7 @@
     "restart": "restart",
     "rollback": "rollback",
     "running resources": "running resources",
-    "number of application instance": "number of application instance",
+    "runtime-resource-config-form-tip": "runtime-resource-config-form-tip",
```

Comment:
This is incorrect

Reply:
"number of application instance" already exists in this module, "runtime-resource-config-form-tip" is new.